js/main.js

```diff
@@ -0,0 +1,90 @@
+
+var userTitle = document.querySelector('.user-title');
+var form = document.querySelector('form');
+var resultUL = document.querySelector('.result-box');
+var views = document.querySelectorAll('.view');
+var home = document.querySelector('.home');
+
+home.addEventListener('click', function () {
+  viewSwitch('search-form');
+});
+
+function saveInfo(event) {
+  event.preventDefault();
+  while (resultUL.firstElementChild) {
+    resultUL.firstElementChild.remove();
+  }
+  data.tempTitle = userTitle.value;
+  searchRequest(data.tempTitle);
+  viewSwitch('search-result');
+}
+
+form.addEventListener('submit', saveInfo);
+
+function createSearch(result) {
```

Comment:
This function should be renamed, it sounds like it's going to send a network request for a search. Maybe rename to `generateSearchResult`